templates/events/event_list.html

```diff
@@ -57,6 +57,26 @@ <h3 class="event-title"><a href="{{ object.get_absolute_url }}">{{ object.title|
                 </ul>
             </div>
 
+            {% if events_ongoing %}
+            <h3 class="widget-title">Ongoing Events</h3>
```

Comment:
```suggestion
            <h3 class="widget-title">Happening Now<h3>
```
I feel like this is "cooler" but just a nit :)